messaging/src/main/java/io/lantern/messaging/metadata/Metadata.kt

```
@@ -77,21 +76,28 @@ class Metadata(
         }
 
         private fun visualMetadata(file: File, mimeType: String?): Metadata {
+            val retriever = MediaMetadataRetriever()
             val bmp = when {
                 mimeType?.startsWith("image") == true -> rotatedBitmap(file)
                 mimeType?.startsWith("video") == true -> {
-                    val retriever = MediaMetadataRetriever()
                     try {
                         retriever.setDataSource(file.absolutePath)
                         retriever.getFrameAtTime(0)
-                    } finally {
-                        retriever.release()
+                    } catch (t: Throwable) {
+                        null
```

Why swallow the exception here and on line 95?